boards/arm/rak5010_nrf52840/rak5010_nrf52840.dts

```diff
@@ -75,7 +75,7 @@
 };
 
 &uart1 {
-	compatible = "nordic,nrf-uart";
+	compatible = "nordic,nrf-uarte";
```

Comment:
It would be better just to remove this line completely. The peripheral does not have the option to be reconfigured as UART, so just let nrf52840.dtsi provide the correct value for compatible.